scalableminds/webknossos
# Attach action log to update action batches

Attach a log of up to 50 redux actions to each version when saving for debugging purposes. This will allow us to see the latest actions a user took to produce a specific tracing version.

@fm3 The update action batches now have an additional `info` string property (next to version and stats, etc). Could you make sure to persist this property and return it when requesting the updateActionLog?

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Modify the save_saga locally and set compress to false.
- Open a tracing and open the network tab. Move and save - the update request should contain update action batches with the `info` property that contains the latest redux actions.

### Issues:
- will help with debugging the inconsistent tracing states

------
